CardData.py

```diff
@@ -92,93 +96,91 @@ def create_card_json(gwent_data_helper, patch):
-        if key[-1] == "_":
-            key = key[:-1]
+        card_id = template.attrib['Id']
+        card['ingameId'] = card_id
+        card['strength'] = int(template.find('Power').text)
+        card['type'] = TYPES.get(int(template.find('Tier').text))
+        card['faction'] = FACTIONS.get(int(template.find('FactionId').text))
 
         card['name'] = {}
         card['flavor'] = {}
         for region in GwentUtils.LOCALES:
-            card['name'][region] = gwent_data_helper.card_names.get(region).get(key)
-            card['flavor'][region] = gwent_data_helper.flavor_strings.get(region).get(key)
+            card['name'][region] = gwent_data_helper.card_names.get(region).get(card_id)
+            card['flavor'][region] = gwent_data_helper.flavor_strings.get(region).get(card_id)
 
         # False by default, will be set to true if collectible or is a token of a released card.
         card['released'] = False
 
-        if template.find('Tooltip') is not None:
-            tooltip_id = template.find('Tooltip').attrib['key']
-            card['info'] = {}
-            card['infoRaw'] = {}
-            for locale in GwentUtils.LOCALES:
-                tooltip = gwent_data_helper.tooltips[locale].get(tooltip_id)
-                if tooltip is not None:
-                    card['infoRaw'][locale] = tooltip
-                    card['info'][locale] = GwentUtils.clean_html(tooltip)
+        card['info'] = {}
+        card['infoRaw'] = {}
+        for locale in GwentUtils.LOCALES:
+            tooltip = gwent_data_helper.tooltips[locale].get(card_id)
+            if tooltip is not None:
+                card['infoRaw'][locale] = tooltip
+                card['info'][locale] = GwentUtils.clean_html(tooltip)
 
-            card['keywords'] = gwent_data_helper.keywords.get(tooltip_id)
+        card['keywords'] = gwent_data_helper.keywords.get(card_id)
 
-        card['positions'] = []
+        card['positions'] = ["Melee", "Ranged", "Siege"]
         card['loyalties'] = []
-        for flag in template.iter('flag'):
-            key = flag.attrib['name']
-
-            if key == "Loyal" or key == "Disloyal":
-                card['loyalties'].append(key)
-
-            if key == "Melee" or key == "Ranged" or key == "Siege" or key == "Event":
-                card['positions'].append(key)
+        placement = template.find('Placement')
+        if placement.attrib['PlayerSide'] != "0":
+            card['loyalties'].append("Loyal")
+        if placement.attrib['OpponentSide'] != "0":
+            card['loyalties'].append("Disloyal")
 
         card['categories'] = []
-        for flag in template.iter('Category'):
-            key = flag.attrib['id']
-            if key in CATEGORIES:
-                card['categories'].append(CATEGORIES.get(key))
 
         card['variations'] = {}
-
-        for definition in template.find('CardDefinitions').findall('CardDefinition'):
+        for x in range(1):
```

Comment:
???

Reply:
Yeah I'll fix this. This is the result of some panicked coding to get a working JSON as soon as possible on the night of the patch.

Variations no longer exist in Gwent either. To support our current data format, I've set each card to have 1 variation.